dietpi/dietpi-software

```diff
@@ -12236,7 +12253,33 @@ _EOF_
 			fi
 
 		fi
+		
+		software_id=185 # Portainer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
 
+			# Restart Docker service to be able to deploy the container
+			G_DIETPI-NOTIFY 2 'Docker will be restarted to be able to deploy the container.'
+			G_EXEC systemctl daemon-reload
+			G_EXEC systemctl restart docker.service
```

Comment:
The `.service` ending is btw optional, as it is assumed automatically by systemctl.

Reply:
ok will remove `.service`